connector/connect/src/main/scala/org/apache/spark/sql/connect/config/Connect.scala

```diff
@@ -26,4 +26,12 @@ private[spark] object Connect {
       .intConf
       .createWithDefault(15002)
 
+  val CONNECT_GRPC_INTERCEPTOR_CLASSES =
```

Comment:
Just realized that this is under `apach.spark.sql.` ... we should either move this module out of `sql` or use `StaticSQLConf` ideally.

Reply:
This is in the connect module.

Reply:
but it's under `apache.spark.sql.connect`. Should probably move it to `apache.spark.connect`?

Reply:
right now, everything in Spark connect is under `apache.spark.sql.connect`. Are you proposing a overall package renaming?

Reply:
Yeah, I am proposing before it's too late.

Either: if we target to cover other components too, should probably rename them before it's too late. For PySpark too, should probably move it from `pyspark.sql.connect.DataFrame` -> `pyspark.connect.sql.DataFrame`.

Or, use `StaticSQLConf` since we're in SQL package. We're doing this in Hive thirft server, Hive modules, etc.

Reply:
The reason for being under sql is that we need to access some package protected functionality from catalyst. In addition, the intention was that it's clear that it's built on top of the sql functionality.

For the clients it's a different thing. The clients should Ideally have the exact same package names as our client APIs have today.

Reply:
Spark connect is really a SQL thing (DF API, no RDD support). The current package name looks fine.

Reply:
Then we should better use `StaticSQLConf` or `SQLConf` instead of `SparkConf`.

Reply:
My point is that it's mixed. It's in SQL package but the configuration being used is `SparkConf`.

Reply:
StaticSQLConf and SQLConf are in the SQL module, it's weird to add spark connect configs there...

Reply:
That's what Hive thriftserver (separate module) dose. Avro (separate module) and Kafka (separate module for Structured Streaming) do. Pandas API on Spark also leverages runtime configurations via SparkSession under the hood instead of SparkConf.

This is weird that it's a SQL thing uses SQL package namespace but it doesn't use `SQLConf`.

Reply:
I'm not sure what's the benefit of doing so. Config definition can be anywhere and we can still use SQLConf to access it. e.g. `SQLConf.get.getConf(CONNECT_GRPC_INTERCEPTOR_CLASSES)`.